spec/integration/store/TimelineSpace/HeaderMenu.spec.js

```diff
@@ -0,0 +1,70 @@
+import Mastodon from 'megalodon'
+import { createLocalVue } from '@vue/test-utils'
+import Vuex from 'vuex'
+import { ipcMain } from '~/spec/mock/electron'
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Error - 'ipcMain' is defined but never used. (no-unused-vars)